test/powershell/Host/TabCompletion/TabCompletion.Tests.ps1

```diff
@@ -480,8 +480,8 @@ Describe "TabCompletion" -Tags CI {
                 $beforeTab = 'filesystem::{0}\Wind' -f $env:SystemDrive
                 $afterTab = 'filesystem::{0}\Windows' -f $env:SystemDrive
             } else {
-                $beforeTab = 'filesystem::/us' -f $env:SystemDrive
-                $afterTab = 'filesystem::/usr' -f $env:SystemDrive
+                $beforeTab = 'filesystem::/sbi' -f $env:SystemDrive
+                $afterTab = 'filesystem::/sbin' -f $env:SystemDrive
```

Comment:
`$env:SystemDrive` plays no role here and isn't even defined on non-Windows, so that whole part should be removed.  Seems like maybe we should use the testdrive for the filesystem path as something predictable and can be used on all systems.  The `Windows` folder isn't anything special here with regards to tab completion.

Reply:
Updating this test

Reply:
Formatting doesn't work in the test at all and we should remove it. I believe we leave this "as is" now and fix the test later.
Tracking Issue #6739